src/next-app/pages/quote-request.tsx

```diff
@@ -0,0 +1,328 @@
+                    </div>
+                    : null
+                }
+                {make && make !== 'Other' ?
+                    <div className={styles['text-container']}>
+                        <SingleDropdownField name="Select Vehicle Model" items={models[make]} onChange={setModel} required />
+                    </div>
+                    : null
+                }
+                {model || customModel ?
+                    <div className={styles['text-container']}>
+                        <TextField name="Vehicle Model Year" placeholder="Enter vehicle model year" onChange={setModelYear} required inputType='number' width={'25%'} />
+                    </div> : null
+                }
+            </div>
+            <div className={styles.subtitle}>
+                <label>
+                    Contact Information
+                </label>
+            </div>
+            <div className={styles.content}>
+                <div className={styles['text-container']}>
+                    <TextField name="First Name" placeholder="Enter your first name" onChange={setFirstName} required />
+                </div>
+                <div className={styles['text-container']}>
+                    <TextField name="Last Name" placeholder="Enter your last name" onChange={setLastName} required />
+                </div >
+                <div className={styles['text-container']}>
+                    <TextField name="Phone Number" placeholder="Enter your phone number" onChange={setPhoneNumber} required />
+                </div>
+                <div className={styles['text-container']}>
+                    <TextField name="Email" placeholder="Enter your email" onChange={setEmail} onBlur={handleEmailBlur} errors={emailErrors} required />
+                </div>
+                <div className={styles['text-container']}>
+                    <SingleDropdownField name="Preferred Contact Method" items={['Email', 'Phone']} onChange={setPreferredContact} />
+                </div>
+            </div>
+            <div className={styles.subtitle}>
+                <label>
+                    Quote Request Information
+                </label>
+            </div>
+            <div className={styles.content}>
+                <div className={styles['text-container']}>
+                    <SingleDropdownField name="New or Used Part Preference" items={['New Parts Only', 'Used Parts Only', 'No Preference']} onChange={setPartPreferenceSeller} />
+                </div>
+                <div className={styles['text-container']}>
+                    <SingleDropdownField name="OEM or Aftermarket Part Preference" items={['OEM Parts Only', 'Aftermarket Parts Only', 'No Preference']} onChange={setPartPreferenceType} />
+                </div>
+                <div className={styles['text-container']}>
+                    <TextArea name='Additional Notes' placeholder='Enter any additional notes here.' onChange={setNotes} />

```

Comment:
Instead of letting the user resize this, we will need to make a component that adjusts its height dynamically by itself based on the size of its content.